### ISSUE TITLE
ritual of the harvest moon ( blood magic ) isn't working with magical crops 

### ISSUE THREAD

**Issue by deadpool772** [WTFGeeks/Natural-Magic#130]
The ritual works with Pam's harvest craft but not magical crops Idk if this is planned or being sorted out

**Comment by bolezub** [comment]
I have found out the same thing. Had to use thaumcraft golems instead.

**Comment by Dranyth** [comment]
Interesting... I actually didn't notice that Ritual of the Green Grove wasn't working with Magical Crops because the way I used it was by making an 11x11 farm plot with a 3x3 hole in the middle of it.  In the 3x3 I put the Ritual of the Green Grove one block down, put water on top of that, and 9 Lilypads of Fertility on the water.  The Ritual causes the Lilypads to apply their growth ticks faster, which also affect the other Lilypads as well as any other plants around them.  Any magical crop I plant grows in the matter of 5-8 seconds for the most part.

**Comment by Pokecraft12346** [comment]
Yeah, magical crops is quite picky with what it works with. I've found that out of all of the tech AND magic mods from most modpacks, only thaumcraft golems seem to work... It's really weird.

But you have to admit. Golem's farming diamonds is pretty cool. =D
